apps/etterlatte-saksbehandling-ui/client/src/components/behandling/Behandling.tsx

```diff
@@ -34,13 +41,39 @@ export const Behandling = () => {
     }
   }, [behandlingIdFraURL, behandling?.id])
 
+  useEffect(() => {
+    resetSjekklisteResult()
+    resetOpprettSjekkliste()
+    if (behandling && isInitial(hentSjekklisteResult)) {
+      hentSjekklisteForBehandling(
+        behandling.id,
+        (result) => {
+          dispatch(updateSjekkliste(result))
+        },
+        () => {
+          if (!erFerdigBehandlet(behandling.status)) {
+            opprettSjekklisteForBehandling(behandling.id, (opprettet) => {
+              dispatch(updateSjekkliste(opprettet))
+            })
+          }
+        }
+      )
+    }
+  }, [behandling])
```

Comment:
Flyttet dette hit, etter opprinnelig hatt det i sjekkliste-fanen. Problemet med det er da at valideringen ved "send til attestering" kan trigge uten at sjekklista er lastet, og dermed ikke medføre noe valideringsfeil. Tar gjerne imot innspill på bedre løsning.

Uansett åpen for at selve opprettelsen kanskje heller burde skje i backend når en behandling opprettes.